documentation/modules/ROOT/pages/connectors/sqlserver.adoc

```diff
@@ -103,9 +145,16 @@ The connector is able to detect whether CDC is enabled or disabled for included
 [[sqlserver-topic-names]]
 === Topic names
 
-The SQL Server connector writes events for all insert, update, and delete operations on a single table to a single Kafka topic. The name of the Kafka topics always takes the form _serverName_._schemaName_._tableName_, where _serverName_ is the logical name of the connector as specified with the `database.server.name` configuration property, _schemaName_ is the name of the schema where the operation occurred, and _tableName_ is the name of the database table on which the operation occurred.
+The SQL Server connector writes events for all `INSERT`, `UPDATE`, and `DELETE` operations on a single table to a single Kafka topic. 
+The name of the Kafka topics always takes the form _serverName_._schemaName_._tableName_.
```

Comment:
```suggestion
The name of the Kafka topics by default takes the form _serverName_._schemaName_._tableName_.
```

(the logic table router can adjust this)

Reply:
Is this configured through [Custom topic auto creation]https://debezium.io/documentation/reference/1.3/configuration/topic-auto-create-config.html?

Reply:
@gunnarmorling IMHO the naminig always takes the described form unless an SMT is used. Now the wording implies that it is something that can be change by configuration.

Reply:
> Is this configured through Custom topic auto creation?

I meant the logical topic router, see here: https://debezium.io/documentation/reference/1.3/configuration/topic-routing.html. So it should be something like this:

> The name of the Kafka topics by default takes the form _serverName_._schemaName_._tableName_. If this default isn't suitable for your requirements, you can customize the topic naming by using the logical topic routing SMT (link)...